mobile/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "@textile/go-mobile",
+  "description": "Textile iOS and Android bindings.",
+  "version": "0.1.5",
+  "author": "textile.io",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/textileio/textile-go"
+  },
+  "homepage": "https://www.textile.io/",
+  "keywords": [
+    "textile",
+    "decentralized",
+    "mobile"
+  ],
+  "files": [
+    "scripts"
+  ],
+  "scripts": {
+    "postinstall": "./scripts/fetch_release.sh 0.1.5"
```

Comment:
We should just use `$npm_package_version` here so you don't ever have to update that

Reply:
oh nice, didnt know about that

Reply:
Think it works, def test it haha

Reply:
yep, works